apps/docs/src/.vitepress/components/TopicOverviewCard.vue

```diff
@@ -0,0 +1,75 @@
+<script lang="ts" setup>
+const props = defineProps<{
+  title: string;
+  subtitle: string;
+  image: string;
+  href: string;
+}>();
+</script>
+
+<template>
+  <a class="card" tabindex="0" :href="props.href">
```

Comment:
```suggestion
  <a class="topic" tabindex="0" :href="props.href">
```

yes, same as last time with the component card: both are cards, but from the semantics, they represent different things ^^